docs/csharp/language-reference/keywords/interpolated-strings.md

```diff
@@ -19,12 +19,12 @@ Used to construct strings.  An interpolated string looks like a template string
 The arguments of an interpolated string are easier to understand than a [composite format string](../../../standard/base-types/composite-formatting.md#composite-format-string).  For example, the interpolated string  
   
 ```csharp  
-Console.WriteLine($"Name = {name}, hours = {hours:hh}"); 
+Console.WriteLine($"Name = {name}, hours = {hours}:hh"); 
 ```  
 contains two interpolated expressions, '{name}' and '{hours:hh}'. The equivalent composite format string is:
 
 ```csharp
-Console.WriteLine("Name = {0}, hours = {1:hh}", name, hours);  
+Console.WriteLine("Name = {0}, hours = {1}:hh", name, hours);  
```

Comment:
I think the original code is correct: the `:hh` is a format string, not something that should be printed.

But the original code is also confusing: if it's supposed to print hours, then `hours` should be a `DateTime`. But `hours` is not a good name for a `DateTime` variable.

I would change the snippets to:

```c#
Console.WriteLine($"Name = {name}, hours = {date:hh}"); 
Console.WriteLine("Name = {0}, hours = {1:hh}", name, date);  
```

Reply:
you are right!

Reply:
@iubns Do you want to update this PR per @svick's comments?  If so, just add new commits and sync it again.

Thanks for the contribution. We really appreciate it.